preview-src/cache.ts

```diff
@@ -38,6 +38,14 @@ export function getState(): PullRequest {
 }
 
 export function setState(pullRequest: PullRequest): void {
+	let oldPullRequest = getState();
```

Comment:
Could we just save the pendingCommentText as the draft for that comment? It looks like then it'll get restored automatically?

Reply:
It's for the content in the bottom textarea, which is not bind to any comment yet.